specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json

```diff
@@ -903,7 +917,7 @@
             "HealthcareProfession",
             "Diagnosis",
             "SymptomOrSign",
-            "ConditionalQualifier",
+            "ConditionQualifier",
```

Comment:
Was this what the service sent back? This was a GA'd enum value, wasn't it? We'll have to take special precaution in the SDK. /cc @kristapratico @deyaaeldeen @joseharriaga @mssfang

Reply:
I don't have a test that covers this case but both the v3.1 and 2022-05-01 swaggers have "ConditionQualifier"

Reply:
Please be sure whatever you're putting here matches the actual service behavior